app/experimenter/nimbus-ui/src/components/TableHighlights/index.tsx

```diff
@@ -50,13 +50,17 @@ const TableHighlights = ({
         {Object.keys(results).map((branch) => {
           const userCountMetric =
             results[branch]["branch_data"][METRIC.USER_COUNT];
+          const participantCount =
+            userCountMetric[BRANCH_COMPARISON.ABSOLUTE].length > 0
+              ? userCountMetric[BRANCH_COMPARISON.ABSOLUTE][0]["point"]
+              : null;
           return (
             <tr key={branch} className="border-top">
               <th className="align-middle p-1 p-lg-3" scope="row">
                 <p>{branch}</p>
                 <p className="h6">
-                  {userCountMetric[BRANCH_COMPARISON.ABSOLUTE]["point"]}{" "}
-                  participants ({userCountMetric["percent"]}%)
+                  {participantCount} participants ({userCountMetric["percent"]}
```

Comment:
Q: Above on L56 we set `participantCount` to `null` if the `userCountMetric[BRANCH_COMPARISON.ABSOLUTE]` array is empty.

How/what would this render if the `participantCount` is null?

Reply:
I've updated the server-side generated data so this is not the case anymore